README.md

```diff
@@ -274,7 +274,7 @@ docsSearchBar({
 
 Here is a basic [HTML file](playground/index.html) used in the playground of this repository.
 
-As a more concrete example, you can [check out the configuration](https://github.com/meilisearch/vuepress-plugin-meilisearch/blob/bf2210e131eb5630f632d514f35710ee6576ec7a/MeiliSearchBox.vue#L64) applied in the MeiliSearch plugin for VuePress.
+As a more concrete example, you can [check out the configuration](https://github.com/meilisearch/vuepress-plugin-meilisearch/blob/bf2210e131eb5630f632d514f35710ee6576ec7a/MeilisearchBox.vue#L64) applied in the Meilisearch plugin for VuePress.
```

Comment:
I just try this new link and it's broken ... Apparently, GitHub URLs are Case Sensitive?

Reply:
Good point @alallema but it breaks because it is related to a real "file" inside the repo and they are case sensitive.

But it shouldn’t be: `https://github.com/meilisearch/vuepress-plugin-meilisearch/blob/main/MeiliSearchBox.vue#L69` (pointing to main directly? @bidoubiwa)

Reply:
Yes absolutely ! Im changing that :)